### ISSUE TITLE
Initial load doesn't have base styles

### ISSUE THREAD

**Issue by nolawi** [department-of-veterans-affairs/caseflow#6864]
With the introduction of hashed css files, initial load doesn't have any cached styles. Its nice for local/dev env but if it happens in prod after every deploy, it looks terrible

![2018-09-06 19 49 15](https://user-images.githubusercontent.com/1745984/45223546-8d8ca780-b285-11e8-9766-7d5cdc5d898a.gif)

1. We should consider having a 'normalize' like base styles, outside of webpack 
2. OR, webpack lazy load

**Comment by nolawi** [comment]
@mdbenjam let me me know if you have any thoughts here

**Comment by mdbenjam** [comment]
Yes I've noticed this as well. I'm not familiar with either of these solutions, could you elaborate further? I'd love to fix this.

**Comment by nolawi** [comment]
I think, initial load of our app will everything together, markup shows up before browser can translate the styles. Since the css url is always new on every deploy, it will never use any cached styles hence we see unstyled markup first. We never had an issue before because styles were loaded before. I think wepack solves this by breaking it up in multiple webpack files. https://webpack.js.org/guides/lazy-loading/#example. 

Or very simpler solution would be to just have base cached styles in the rails.
